SeeSharp/Geometry/MeshLoadException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SeeSharp.Geometry {
+    public class MeshLoadException : Exception {
```

Comment:
This should be documented with an XML comment (also the Path property and the constructors).